robot/description/urdf/inertias.xacro

```diff
@@ -52,5 +52,18 @@
             izx="0.0" izy="0.0" izz="${0.5*mass*radius*radius}"
         />
     </xacro:macro>
+    <!-- Camera -->
+    <xacro:macro name="box_inertia" params="mass length width height">
+        <inertia
+            ixx="${mass * (height*height + length*length)/12.0}" 
+                        ixy="0.0" 
+            ixz="0.0" 
+                        iyy="${mass * (width*width + length*length)/12.0}" 
+                        iyz="0.0" 
+            izz="${mass * (width*width + height*height)/12.0}"
+        />
+                
+    </xacro:macro>
+    
```

Comment:
We should not need to add a new "box_inertia", it has already been added from the GPS PR and used by the IMU PR. Likewise, the spacing is weird for the inertia macro (I fixed this spacing earlier when I merged the GPS plugin PR)